opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -270,12 +270,14 @@ class ConsoleSpanExporter(SpanExporter):
     def __init__(
         self,
         out: typing.IO = sys.stdout,
-        formatter: typing.Callable[[Span], str] = str,
+        formatter: typing.Callable[[Span], str] = lambda span: str(span)
+        + "\n",
```

Comment:
```suggestion
        formatter: typing.Callable[[Span], str] = str,
```

Along with the other comment suggestion only.